docs/gen-qa-openai.ipynb

```diff
@@ -412,173 +454,328 @@
     "id": "kXXYoANbG13a"
    },
    "source": [
-    "To search through our documents we first need to create a query vector `xq`. Then, using `xq` we will retrieve the most relevant chunks from our index. \n",
+    "To implement cascading retrieval, we will need to conduct the following steps:\n",
+    "\n",
+    "1. Query: We query our sparse and dense indexes with a large top_k value\n",
+    "2. Deduplicate: We combine and deduplicate results from the previous queries to achieve a preliminary result set\n",
+    "3. Rerank: We pass this result set, and the query, to a Rerank endpoint hosted on Pinecone, returning our final set of documents\n",
+    "\n",
+    "The final set of documents can be considered the most relevant queries across dense and sparse search methods for the user.\n",
     "\n",
-    "To create that query vector we will again use OpenAI's `text-embedding-ada-002` model. For this, you need an [OpenAI API key](https://platform.openai.com/)."
+    "To learn more about exactly how reranking models can help improve relevance of search results, take a [look at our article here](https://www.pinecone.io/learn/refine-with-rerank/)."
```

Comment:
This is minor, but can we move away from ableist language like this? There are a few instances throughout, but you could use "check out X" or "refer to Y" or "build an ZYZ" or "follow these instructions to do X"

Reply:
Sure! I'll replace the verbiage no problem!